evalml/tests/pipeline_tests/test_time_series_pipeline.py

```diff
@@ -1060,6 +1060,92 @@ def test_binary_predict_pipeline_objective_mismatch(
         binary_pipeline.predict(X[30:32], "precision micro", X[:30], y[:30])
 
 
+@pytest.mark.parametrize(
+    "problem_type",
+    [
+        ProblemTypes.TIME_SERIES_BINARY,
+        ProblemTypes.TIME_SERIES_MULTICLASS,
+        ProblemTypes.TIME_SERIES_REGRESSION,
+    ],
+)
+def test_time_series_pipeline_no_ytrain_predict_error(
+    problem_type,
+    ts_data_binary,
+    ts_data_multi,
+    ts_data,
+    time_series_binary_classification_pipeline_class,
+    time_series_multiclass_classification_pipeline_class,
+    time_series_regression_pipeline_class,
+):
+    if problem_type == ProblemTypes.TIME_SERIES_BINARY:
+        X, y = ts_data_binary
+        clf = time_series_binary_classification_pipeline_class(
+            parameters={
+                "Logistic Regression Classifier": {"n_jobs": 1},
```

Comment:
if all the parameters are the same for each of these pipelines, we can save the parameters up top and then call it to save some space.